### PR TITLE
Prepare 3.3.2 release

### PR DESCRIPTION
**Pull request by emilywoods**

### CODE REVIEW

File: blackbox/docs/appendices/release-notes/3.3.2.rst [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+:local:
+
+Changelog
+=========
+
+Fixes
+-----
+
+- Fixed the processing of ``LIMIT`` clauses within queries in the :ref:`INSERT
+    INTO <ref-insert>` statement. A query like ``INSERT INTO target (SELECT *
```
Comment (christianbader) [inline blackbox/docs/appendices/release-notes/3.3.2.rst]: Fix indent please.

File: blackbox/docs/appendices/release-notes/unreleased.rst [hunk 2]
```diff
@@ -50,19 +50,3 @@ None
...
-  In addition, using ``LIMIT`` in the top level query of the ``INSERT INTO``
-  statement is now no longer prohibited. So the query can be written as
-  follows: ``INSERT INTO target (SELECT * FROM source LIMIT 10)``.
-
-- Fixed an issue that would cause the wrong evaluation of nested sub-queries
-  in cases where the inner sub-query returns a multi-value and the outer returns
-  a single value result. For instance, the assignment sub-query expression in
-  the following update statement
-  ``UPDATE t1 SET x = (SELECT COUNT(*) FROM t2 WHERE x IN (SELECT x FROM t3))")``
-  might have produced an incorrect result.
```
[the remark this answers is not in the document]
Comment (emilywoods) [reply blackbox/docs/appendices/release-notes/unreleased.rst]: Done ✅

File: blackbox/docs/appendices/release-notes/3.3.2.rst (hunk 1 above)
Comment (emilywoods) [reply blackbox/docs/appendices/release-notes/3.3.2.rst]: oops, fixed ✅

**Comment by christianbader** [comment]
retest this please

**Comment by christianbader** [comment]
retest this please

**Pull request merged**